pinot-common/src/main/java/com/linkedin/pinot/common/protocols/SegmentCompletionProtocol.java

```diff
@@ -233,12 +234,21 @@ public long getBuildTimeMillis() {
       public long getWaitTimeMillis() {
         return _waitTimeMillis;
       }
-      public int getExtTimeSec() {
-        return _extTimeSec;
+      public int getExtraTimeSec() {
+        return _extraTimeSec;
       }
     }
   }
 
+  public static class ExtendBuildTimeRequest extends Request {
+    public ExtendBuildTimeRequest(Params params) {
+      super(params);
+    }
+    @Override
+    public String getUrl(final String hostPort) {
+      return "http://" + hostPort + getUri(MSG_TYPE_EXTEND_BUILD_TIME);
```

Comment:
Can we have a default implementation of this, it looks they're all the same except for the message type anyway.